src/directives/sortable.js

```diff
@@ -86,7 +89,7 @@ ngeo.sortableDirective = function($timeout) {
             }
 
             if (!goog.isNull(dragListGroup)) {
-              goog.events.removeAll(dragListGroup);
+              dragListGroup.dispose();
```

Comment:
No, and using `dragListGroup.dispose()` may be better, because `gooo.dispose`  is not strict with types, and [does](https://github.com/google/closure-library/blob/master/closure/goog/disposable/disposable.js#L279-L288) runtime checks on the object passed.